doc_template/examples_root/examples/nb/visual_behavior_ophys_dataset_manifest.ipynb

```diff
@@ -59,172 +59,81 @@
      "name": "stdout",
```

Comment:
<p>Looks good. I would ask that you explain the logic a bit better for the n_cells per experiment calculation. I understand the calculation, but the <code>to_frame</code> and <code>rename</code> make it a bit hard to parse. Maybe store the intermediate result first showing the calculation and put inline comments  to explain what <code>to_frame</code> and <code>rename</code> are for.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/AllenSDK/pull/2620/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='169'/>

Reply:
I simplified the expression